nvflare/private/fed/app/server/runner_process.py

```diff
@@ -148,4 +148,5 @@ def main():
     This is the program when starting the child process for running the NVIDIA FLARE server runner.
     """
     # main()
-    mpm.run(main_func=main)
+    rc = mpm.run(main_func=main)
```

Comment:
What is the return code if MPM can't finished cleanup in given time?

Reply:
Added additional FINISHED_ABNORMAL for this particular can't finished cleanup in given time scenario. The return code is -9.

Reply:
got it, sounds good.